### PR TITLE
[ADD] pylint-odoo: Adding new check unnecessary-attribute-noupdate

### PR DESCRIPTION
**Pull request by JesusZapata**
This check detected the unnecessary noupdate attribute `<odoo noupdate=0>/<data noupdate=0>` because the default value of attribute no update is zero

Example of the detected code 

```xml
<?xml version="1.0" encoding="utf-8"?>                                          
<odoo noupdate="0">
    ...
</odoo>    
```
```xml
<?xml version="1.0" encoding="utf-8"?>                                          
<odoo>
    <data noupdate="0">
    ...
    </data>
</odoo>    
```
Fixes https://github.com/Vauxoo/pylint-odoo/issues/119

### CODE REVIEW

File: pylint_odoo/checkers/modules_odoo.py [hunk 1]
```diff
@@ -756,6 +762,25 @@ def _check_deprecated_data_xml_node(self):
...
+        xml_files = self.filter_files_ext('xml')
+        self.msg_args = []
+        for xml_file in xml_files:
+            doc = self.parse_xml(os.path.join(self.module_path, xml_file))
+            nodes = []
+            for xpath in ("/odoo/data[@noupdate='0']", "/odoo[@noupdate='0']"):
+                nodes.extend(doc.xpath(xpath)
+                             if not isinstance(doc, basestring) else [])
+            if nodes:
+                lineno = nodes[0].sourceline
```
Comment (moylop260) [inline pylint_odoo/checkers/modules_odoo.py]: I didn't understand why you are appending nodes to just use the first one

Comment (JesusZapata) [inline pylint_odoo/checkers/modules_odoo.py]: @moylop260 
Sorry for my bad copy & paste

**Comment by JesusZapata** [comment]
@moylop260 
What do you think of this PR?

**Comment by moylop260** [comment]
Could you migrate this PR to OCA project, please?

**Comment by JesusZapata** [comment]
@moylop260 
The related pull request in OCA is https://github.com/OCA/pylint-odoo/pull/137

**Comment by JesusZapata** [comment]
@moylop260 
I add one file with two tag `noupdate` into the same file

Similar to 

```xml
<?xml version="1.0" encoding="utf-8"?>
<odoo>
    <data noupdate="0">
    </data>
    ...
    <data noupdate="1">
    </data>
</odoo>
```

Related with your comment https://github.com/OCA/pylint-odoo/pull/137#issuecomment-316423473

**Comment by luisg123v** [comment]
OCA PR was closed, on https://github.com/OCA/pylint-odoo/pull/137#issuecomment-468055919, hence I'm doing the same for this one.

**Pull request closed without merging**